layouts/sitemap.xml

```diff
@@ -0,0 +1,21 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  {{ range .Data.Pages }}{{ if ne .Params.sitemapExclude true }}
+  <url>
+    <loc>{{ .Permalink }}</loc>{{ if not .Lastmod.IsZero }}
+    <lastmod>{{ safeHTML ( .Lastmod.Format "2006-01-02T15:04:05-07:00" ) }}</lastmod>{{ end }}{{ with .Sitemap.ChangeFreq }}
+    <changefreq>{{ . }}</changefreq>{{ end }}{{ if ge .Sitemap.Priority 0.0 }}
+    <priority>{{ .Sitemap.Priority }}</priority>{{ end }}{{ if .IsTranslated }}{{ range .Translations }}
+    <xhtml:link
+                rel="alternate"
+                hreflang="{{ .Lang }}"
+                href="{{ .Permalink }}"
+                />{{ end }}
+    <xhtml:link
```

Comment:
Why is this a duplicate of line 9 onwards?

Reply:
this is used to list links to the same page in different languages (should they exist)

Reply:
Still not quite getting it. 
First a loop for all translated pages and then the default page? What happens if the page is not translated (`.isTranslated == false`) from my understanding no link is added it that case.

Reply:
The "normal" URL is stored in `<loc>` and is added for all pages (except those, which exclude themselves).

Reply:
Resolving this, although I am still skeptical